simtools/visualize.py

```diff
@@ -364,14 +373,14 @@ def plot1D(data, **kwargs):
         plt.title(title, y=1.02)
     if "_default" not in list(dataDict.keys()) and not noLegend:
         plt.legend()
-    if not plotRatio:
+    if not plotRatio or plotDifference:
```

Comment:
I am not entirely sure what this tests: is it `not plotRatio` or `not plotDifference` or `(not plotRatio) or plotDifference`? 
(I know that python is probably clear on this, but it is hard to read.

Reply:
This is a mistake, it should be `if not (plotRatio or plotDifference):`